src/Whim.Bar/bar.defaults.xaml

```diff
@@ -0,0 +1,87 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
+
+	<!--  Font family  -->
+	<FontFamily x:Key="_font:family">Segoe UI</FontFamily>
+
+	<!--  Colors: Light mode, TODO: improve  -->
```

Comment:
> But maybe it'd be better to keep the background in `bar.defaults.xaml` unset.

I think this would work better. I admit I am one of the few who changes the mode frequently - I use [Auto Dark Mode](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode) to run light mode during the day, and dark mode at night.